experimental/packages/otlp-exporter-base/src/platform/browser/util.ts

```diff
@@ -24,30 +24,6 @@ import {
   parseRetryAfterToMills,
 } from '../../util';
 
-/**
- * Send metrics/spans using browser navigator.sendBeacon
- * @param body
- * @param url
- * @param blobPropertyBag
- * @param onSuccess
- * @param onError
- */
-export function sendWithBeacon(
```

Comment:
This cannot be removed without a serious discussion and significant user benefit

Reply:
I agree with @dyladan during page unloading `sendBeacon` and `fetch` (with keepalive) are the only options for getting data off the browser. And fetch with keep-alive is not always supported.

Reply:
I checked the compatibility of these options. ~5% of the browsers support `sendBeacon` but not `keepalive`.
Maybe we should implement with `fetch` and use `sendBeacon` as a fallback?

Reply:
Further discussion: #3845